src/js/views/right_panel_view.js

```diff
@@ -777,6 +808,17 @@
             $("#setIdModal .imgId").val("").focus();
         },
 
+        lockAspectRatio: function(event) {
+            event.preventDefault();
+            $(".set_aspect_ratio", this.$el).toggleClass("active");
+            if ($(".set_aspect_ratio", this.$el).hasClass("active")) {
+                $(".aspect_ratio_selected", this.$el).addClass("glyphicon-ok-sign");
+            }
+            if (!$(".set_aspect_ratio", this.$el).hasClass("active")) {
+                $(".aspect_ratio_selected", this.$el).removeClass("glyphicon-ok-sign");
+            }
```

Comment:
If you style the ```. aspect_ratio_selected``` to only be visible when the parent is ```.active``` (see other comment) then you can get rid of all this extra logic.